roles/contiv_network/templates/netplugin.j2

```diff
@@ -0,0 +1,7 @@
+{# If an interface is specified for netplugin, use it #}
+{# Otherwise, hope for the best                       #}
```

Comment:
It might be better to fail ansible instead of leaving the setup in some undesired state?